### PR TITLE
ec2_vol module: add support for name / id to mount volume on instance

### PR DESCRIPTION
**Pull request by ebrevdo**
- volume_size no longer required if name OR id is provided
- id is volume-id
- name is volume Name tag
- special checking is provided

### CODE REVIEW

**Comment by jctanner** [comment]
@ebrevdo can you fix the current merge conflicts so that we can test/review this?

**Comment by ebrevdo** [comment]
Sure... 1 or 2 days.

**Comment by ebrevdo** [comment]
@jctanner Just updated my commit via a force push due to a mistaken rebase earlier this morning.  The change is amended and should be up to date with ansible/devel and ready for testing.  I haven't had a chance to test the merged ec2_vol code.

File: library/cloud/ec2_vol [hunk 1]
```diff
@@ -46,6 +46,18 @@ options:
     required: false
     default: null 
     aliases: []
+  name:
+    description:
+      - volume Name tag if you wish to attach an existing volume (requires instance)
+    required: false
+    default: null
+    aliases: []
+  name:
```
Comment (pas256) [inline library/cloud/ec2_vol]: This should be `id`, right? Otherwise `name` is here twice

Comment (ebrevdo) [inline library/cloud/ec2_vol]: Yup.  Thanks for catching that.  Fixed.

I haven't made any changes in quite some time and hope it's now ready for submission.

**Comment by ebrevdo** [comment]
@pas256 fix is in.  can i help with testing or is it ready to go?

**Comment by pas256** [comment]
@jctanner mentioned a merge conflict earlier. If that is still there, can you fix it please?

**Comment by ebrevdo** [comment]
I fixed it last month.

**Comment by jctanner** [comment]
@ebrevdo thank you!

**Pull request merged**